apps/berlinc/berlin-clock.js

```diff
@@ -88,10 +88,9 @@ Bangle.on('lcdPower', (on) => {
   g.clear();
   if (on) {
     Bangle.drawWidgets();
-    // call your app function here
-    drawBerlinClock();
-  }
-});
+  // call your app function here
+  drawBerlinClock();
```

Comment:
this one looks weird, I'd expect:
```js
  if (on) {
    Bangle.drawWidgets();
    // call your app function here
    drawBerlinClock();
  }
});
```

Reply:
Good catch -- I was doing the eslint automatic fix to [earlier version of this file originally](https://github.com/ikkala/BangleApps/commit/0a32e40b1abdf221b79228a8fb36659c18fa0a00#diff-e593f7fcdc3183a0484d87c313186faa) and after that just blindly auto-merged the newer changes from the author before making todays pull request.

I fix fix this.